django_project/ta_system/models.py

```diff
@@ -198,3 +199,12 @@ class Meta:
         ordering = ('user__last_name',)
         verbose_name = "Student"
         verbose_name_plural = "Students"
+
+
+class SystemStatus(models.Model):
+    status = models.BooleanField(default=False)
+    date_changed = models.DateField(default=now)
```

Comment:
Did you test again to make sure using 'now' is working as expected?

Reply:
now is working properly